crates/bevy_asset/src/reflect.rs

```diff
@@ -271,13 +271,12 @@ mod tests {
         };
 
         let handle = reflect_asset.add(app.world_mut(), &value);
-        let ReflectMut::Struct(strukt) = reflect_asset
+        let strukt = reflect_asset
```

Comment:
"strukt" cracks me up. For other reviewers: you can't call it `struct` because that's a reserved keyword.

Reply:
I would add a comment above this just to be sure
```suggestion
        // struct is a reserved keyword, so we can't use it here
        let strukt = reflect_asset
```